source/common/common/hash.h

```diff
@@ -25,6 +26,14 @@ class HashUtil {
     return XXH64(input.data(), input.size(), seed);
   }
 
+  /**
+   * Return 64-bit hash from the xxHash algorithm.
```

Comment:
This should probably speak to the fact that this hashes a collection of input strings

Reply:
got it